src/test/java/com/conveyal/datatools/editor/controllers/api/EditorControllerTest.java

```diff
@@ -143,6 +147,60 @@ public void canPatchStopsConditionally() throws IOException {
         }
     }
 
+    /**
+     * Test the removal of a stop from stop patterns.
+     */
+    @Test
+    void canRemoveStopFromStopTimesAndPatternStops() throws IOException, SQLException {
+        // Get a fresh feed source so that the editor namespace was updated after snapshot.
+        FeedSource freshFeedSource = Persistence.feedSources.getById(feedVersion.feedSourceId);
+        String stopId = "WARM";
+        String stopCountSql = String.format(
+            "SELECT count(*) FROM %s.stops WHERE stop_id = '%s'",
+            freshFeedSource.editorNamespace,
+            stopId
+        );
+        String stopTimesCountSql = String.format(
+            "SELECT count(*) FROM %s.stop_times WHERE stop_id = '%s'",
+            freshFeedSource.editorNamespace,
+            stopId
+        );
+        String patternStopsCountSql = String.format(
+            "SELECT count(*) FROM %s.pattern_stops WHERE stop_id = '%s'",
+            freshFeedSource.editorNamespace,
+            stopId
+        );
+
+        // Check for presence of stopId in stops.
+        assertThatSqlCountQueryYieldsExpectedCount(stopCountSql ,1);
+        // Check for presence of stopId in stop times.
+        assertThatSqlCountQueryYieldsExpectedCount(stopTimesCountSql ,522);
+        // Check for presence of stopId in pattern stops.
+        assertThatSqlCountQueryYieldsExpectedCount(patternStopsCountSql, 4);
+
+        String path = String.format(
+            "/api/editor/secure/stop/cascadeDeleteStop?stopId=%s&feedId=%s&sessionId=test",
+            stopId,
+            feedVersion.feedSourceId
+        );
+        String response = given()
+            .port(DataManager.PORT)
+            .patch(path)
```

Comment:
Just marking that this would have to be updated if we change the endpoint type

Reply:
Noted and updated to delete.